SparkyFitnessMobile/__tests__/screens/MedicationScheduleFormScreen.test.tsx

```
@@ -0,0 +1,538 @@
+jest.mock('../../src/components/TimeSheet', () => {
+  const React = require('react');
+  const { Pressable, Text } = require('react-native');
+  return {
+    __esModule: true,
+    default: React.forwardRef(
+      ({ onSelectTime }: { onSelectTime: (time: string) => void }, _ref: unknown) => (
+        <Pressable onPress={() => onSelectTime('08:00')}>
+          <Text>mock-time-sheet</Text>
+        </Pressable>
+      ),
+    ),
```

ESLint

[error] Component definition is missing display name

(react/display-name)